src/pkg/caendr/caendr/services/cloud/analytics.py

```diff
@@ -52,6 +61,9 @@ def get_weekly_visits(SERVICE):
   df = pd.DataFrame(out) \
           .sort_values('date') \
           .reindex(['date', 'count'], axis=1)
+  
+  # df.to_csv('archived_ga_data.csv', index=False)
```

Comment:
remove commented out code

Reply:
removed in the commit [4e1e299](https://github.com/AndersenLab/CAENDR/pull/408/commits/4e1e29981e3dad85cd0ec00ef2fbd2610ad5f949)